docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+ARG POSTGRES_VERSION
+ARG POSTGIS_VERSION
+
+FROM postgis/postgis:$POSTGRES_VERSION-$POSTGIS_VERSION
+
+# Configuration Parameters
+LABEL maintainer="MobilityDB Project - https://github.com/MobilityDB/MobilityDB"
+ENV POSTGRES_DB=mobilitydb
+ENV POSTGRES_USER=docker
+ENV POSTGRES_PASSWORD=docker
```

Comment:
> ENV POSTGRES_USER=docker
> ENV POSTGRES_PASSWORD=docker

I do not recommend setting the default password or other weak passwords as default because,
in a cloud environment with default settings, these images are vulnerable. 
There's a high risk of cryptominer or malware infection if the ports are left open.

* https://github.com/docker-library/postgres/issues/1157#issuecomment-1837556142

* https://github.com/search?q=repo%3Adocker-library%2Fpostgres+malware&type=issues

Reply:
Thank you very much for reviewing this PR!
I don't have much experience with docker, so this advice is very much appreciated.

If I understand correctly looking at the [postgres docs](https://github.com/docker-library/docs/blob/master/postgres/README.md#start-a-postgres-instance), I simply remove this from the Dockerfile and let users define their chosen password when running the image?

Reply:
> and let users define their chosen password when running the image?

yes  ..
https://github.com/postgis/docker-postgis?tab=readme-ov-file#usage